tools/dgeni/processors/component-grouper.ts

```diff
@@ -103,6 +107,8 @@ export class ComponentGrouper implements Processor {
         group.additionalInterfaces.push(doc);
       } else if (doc.docType === 'type-alias') {
         group.additionalTypeAliases.push(doc);
+      } else if (doc.docType === 'function') {
```

Comment:
This whole `else if` chain is getting pretty long. Consider using a switch on the `doc.docType`.

Reply:
I'd prefer keeping it like this. Since this is most likely the last `docType` and all the `breaks` don't make the code less or more readable IMO.